cutlass_kernels/CMakeLists.txt

```diff
@@ -43,3 +36,7 @@ set(FPA_INTB_GEMM_SRC
     )
 
 add_library(fpA_intB_gemm SHARED ${FPA_INTB_GEMM_SRC})
+target_compile_options(
+  fpA_intB_gemm PRIVATE
+  $<$<COMPILE_LANGUAGE:CUDA>:-expt-relaxed-constexpr>
+)
```

Comment:
@vinx13 I added a private compile option to target `fpA_intB_gemm` so that it won't be propagated to other targets